graphql2/graph/destinations.graphqls

```diff
@@ -37,6 +37,7 @@ input DestinationFieldSearchInput {
   omit: [String!] # values/ids to omit from results
   after: String # cursor to start search from
   first: Int = 15 # number of results to return
+  favoritesFirst: Boolean = false # sort favorite rotations first
```

Comment:
false is already implied
```suggestion
  favoritesFirst: Boolean # sort favorited items (if supported) first
```

---

That said, is this necessary? Would we ever _not_ want to return favorites first in this context?

I think everywhere we return users we already happen to do fav. first, but in this context it's limited even further to be search-select situations only -- so we can probably leave out the option and just always set favorites first behind the scenes

Reply:
I agree. I've updated it so that we're setting `favoritesFirst: true ` when searching for users, rotations, and schedules in `destinationtypes.go`. Although, since `&graphql2.UserSearchOptions` requires a bool pointer, I had to set `favFirst := true ` on line 128 and use its pointer address, which I suspect isn't best practice. I'd be open to suggestions on how to do this more efficiently.